src/events/normalEvents/messageReactionAdd.ts

```diff
@@ -0,0 +1,62 @@
+import { Event } from "../../utils/Event";
+import { config } from "../../config";
+import type { APIEmbed } from "discord-api-types/v9";
+import { Logger } from "../../utils/Logger";
+
+export const event = new Event(
+	"messageReactionAdd",
+	async (client, reaction, user) => {
+		try {
+			if (
+				!reaction.message.channel.isText() ||
+				reaction.message.channel.id !== config.logs.suggestions ||
+				reaction.message.id === "931837510408089622" ||
+				user.bot
+			)
+				return;
+
+			const channel = client.channels.cache.get(config.logs.suggestions)!;
+
+			if (!channel.isText() || reaction.message.guild == null) return;
+
+			const message = await channel.messages.fetch(reaction.message.id)!;
+
```

Comment:
Check if there are Typescript/Eslint errors and LMK
```suggestion
			const message = await channel.messages.fetch(reaction.message.id)!;

			if (message.channel.id !== config.logs.suggestions || message.guild == null) return;
```